addons/sourcemod/scripting/materialadmin.sp

```diff
@@ -410,10 +411,10 @@ public void Event_PlayerDisconnect(Event eEvent, const char[] sEName, bool bDont
 	if (!GetSteamAuthorized(iClient, sSteamID))
 		return;
 	
-	if (GetUserAdmin(iClient) == INVALID_ADMIN_ID) 
+	if(!(GetUserFlagBits(iClient) & g_iIgnoreFlagOfflineBan))

```

Comment:
This is a “suspicious” code. Even if the player has at least one flag, the condition will work.